### DISCUSSION TITLE
Drag And Drop files

### DISCUSSION THREAD

**Discussion by Icestonks** [dailydotdev/daily#1346] [Feature Requests]
It would be great if we could simply drag and drop or paste files directly into the thumbnail when creating a post.

**Comment by idoshamun** [comment]
Hi @Icestonks,

That's a great suggestion. Thank you!

**Reply by Icestonks** [reply]
Is there any update on this?

**Reply by idoshamun** [reply]
If not mistaken it's already supported. I'll go ahead and close it
